src/framework/cmd/core.c

```diff
@@ -23,8 +23,10 @@ extern void getopt_init(void);
 int cmd_exec(const struct cmd *cmd, int argc, char **argv) {
 	int err;
 
-	if (!cmd)
-		return -EINVAL;
+	if (!cmd) {
```

Comment:
the same question.
Why did you change src/framework/cmd/core.c?
why you could not do it in src/compat/posix/proc/exec.c?

Reply:
It can be done in either place, but I thought it'd be better to do it in the cmd_exec() function as the error is during execution of the cmd, but I can revert back if needed, not a problem :)

Reply:
Reverted